packages/app-project/stores/Organization.js

```diff
@@ -5,9 +5,11 @@ import numberString from './types/numberString'
 const Organization = types
   .model('Organization', {
     id: types.maybeNull(numberString),
+    display_name: types.optional(types.string, ''),
+    listed: types.optional(types.boolean, false),
     primary_language: types.optional(types.string, 'en'),
     slug: types.optional(types.string, ''),
-    strings: types.frozen({}),
+    strings: types.frozen({})
```

Comment:
Translation strings should really be a map of strings: `types.map(types.string)`. I think there are examples of this in the classifier.

Reply:
This is a small enough change that it can go into this PR too.